internal/cli/impl_tui.go

```
@@ -0,0 +1,376 @@
+	ctx context.Context,
+	db db.Handle,
+	cfg core.Config,
+	logsCh <-chan core.LogLine,
+	ids ...core.PMID,
+) error {
+	homeDirOpt := fun.Invalid[string]()
+	if homeDir, err := os.UserHomeDir(); err == nil {
+		homeDirOpt = fun.Valid(homeDir)
+	}
+
+	m := &model{
```

golangci-lint (1.62.2)

cli.model is missing fields dispatch, size, list, keysMap, help, logsList, logTypeShow, logScroll

(exhaustruct)